css/app.css

```diff
@@ -139,21 +138,14 @@ img {
 }
 
 #tabs-list li .close {
-  background-position: center center;
-  background-repeat: no-repeat;
-  display: -webkit-box;
-  margin: 2px 10px 0 0;
-  height: 20px;
-  width: 20px;
+  height: 100%;
+  width: auto;
+  font-size: 20px;
+  visibility: hidden;
```

Comment:
Double checking --- we don't want `display: none`; here? We want this to still remain visible and in need of being laid out by the browser?

Reply:
That's right. Otherwise with long file names the file name will jump to a different length when the close icon becomes visible. Having the file name length remain the same when the close icon appears matches the app's current behaviour, and I think makes the most sense.